signals/subscription_all_categories.py

```diff
@@ -0,0 +1,36 @@
+import swapper
+
+from django.dispatch import receiver
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from django.db.models.signals import (
+    post_save
+)
+
+from categories.models import (
+    UserSubscription,
+    Category
+)
+
+@receiver(post_save, sender = swapper.get_model_name('kernel', 'Person'))
```

Comment:
This model import should be in a separate line. 
`Person = swapper.get_model_name('kernel', 'Person')`
and only Person should be written here.